src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -313,9 +320,11 @@ export class MeetingManager implements AudioVideoObserver {
       this.audioOutputDevices.length
     ) {
       this.selectedAudioOutputDevice = this.audioOutputDevices[0].deviceId;
-      await this.audioVideo?.chooseAudioOutputDevice(
-        this.audioOutputDevices[0].deviceId
-      );
+      if (supportsSetSinkId()) {
+        await this.audioVideo?.chooseAudioOutputDevice(
+          this.audioOutputDevices[0].deviceId
+        );
+      }
```

Comment:
I think this needs a `try..catch` to preserve behavior.

In the v2 branch you need to think very carefully about which failures you want to propagate up to developers and when; you might _not_ `catch` there.